src/lib/components/wizard.component.ts

```diff
@@ -102,6 +162,41 @@ export class WizardComponent implements OnChanges, AfterContentInit {
   @Input()
   public disableNavigationBar = false;
 
+  /**
+   * Navigation custom class
+   */
+  private _navigationCustomClass: string;
+
+  /**
+   * Navigation custom class
+   */
+  @Input()
+  get navigationCustomClass(): string {
+    return this._navigationCustomClass;
+  }
+
+  /**
+   * Navigation custom class
+   */
+  set navigationCustomClass(customClass: string) {
+    delete this._navigationClass[this._navigationCustomClass];
+
+    this._navigationCustomClass = customClass;
+
+    if (customClass) {
+      this._navigationClass[this._navigationCustomClass] = true;
+    }
+  }
+
+  /**
+   * Navigation class
+   */
+  private _navigationClass: any = {};
```

Comment:
Improve performance instead of calculating each time update when change accord

Reply:
Again, I like the idea to create the json object in a typescript method instead of inline in the template.
The creation of the `navigationClass` json object seems to be a bit complex to me.
Do we really require the ability to add a custom class here?
The intended way to change the visual design of the navigation bar is to override the navigation bar style like in https://github.com/madoar/angular-archwizard-demo/blob/develop/src/app/custom-css/custom-css.component.css.